zcash_primitives/src/consensus.rs

```diff
@@ -498,6 +509,7 @@ impl TryFrom<u32> for BranchId {
             0xf5b9_230b => Ok(BranchId::Heartwood),
             0xe9ff_75a6 => Ok(BranchId::Canopy),
             0xc2d6_d0b4 => Ok(BranchId::Nu5),
+            0xc8e71055 => Ok(BranchId::Nu6),
```

Comment:
For clarity, any branch ID we set here is just a placeholder ID, because we only set the final ID (which is always chosen randomly) once the final feature set is implemented and a mainnet activation height is committed to. Also note that if we commit to a mainnet activation height, and then later change it, the branch ID changes (because we change it any time the consensus rules change).